Common/Orders/Slippage/AlphaStreamsSlippageModel.cs

```diff
@@ -23,16 +23,7 @@ namespace QuantConnect.Orders.Slippage
     /// </summary>
     public class AlphaStreamsSlippageModel : ISlippageModel
     {
-        private const decimal _slippagePercent = 0.0001m;
-
-        /// <summary>
-        /// Unfortunate dictionary of ETFs and their spread on 10/10 so that we can better approximate
-        /// slippage for the competition
-        /// </summary>
-        private readonly IDictionary<string, decimal> _spreads = new Dictionary<string, decimal>
-        {
-            {"PPLT",  0.13m}, {"DGAZ", 0.135m}, {"EDV", 0.085m}, {"SOXL", 0.1m}
-        };
+        private const decimal _slippagePercent = 0;
```

Comment:
My mistake; please update to include the same slippage percentage as before, but remove the hardcoded spreads(+hardcode spreads) for the ETFs.